blueprints/page-object/files/tests/pages/__name__.js

```diff
@@ -1,4 +1,4 @@
-import PageObject from '../page-object';
+import PageObject from '<%= testFolderRoot %>/page-object';
```

Comment:
I tested this locally and it seems that we're missing the `/tests` folder in the require statement. It should be something like:

`import PageObject from '<%= testFolderRoot %>/tests/page-object';`

Take a look at https://github.com/ember-cli/ember-cli/blob/2ccd785999f6f815c5738200128a16720bd3f4da/blueprints/acceptance-test/files/tests/acceptance/__name__-test.js#L2